### PR TITLE
Additional FAT tests to validate libraryRef attribute of mpMetrics

### PR DESCRIPTION
**Pull request by Channyboy**
#build

- Tests a non-existant `libraryRef`
- Tests a `libraryRef` with no Micrometer Core Jar (expected to throw errors)
- Tests a `libraryRef` with Micrometer Core + Prometheus Micrometer  Registry + dependencies (can validate that  /metrics endpoint has metrics)
- Tests a `libraryRef` with only Micrometer Core (validates that a SmallRye log is not emitted regarding registering any meter registries to the Micrometer global registry)

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

File: dev/io.openliberty.microprofile.metrics.internal.5.0_fat/fat/src/io/openliberty/microprofile/metrics50/internal/tck/launcher/LibraryRefTest.java [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+        Assert.assertNotNull(line);
+
+        String exceptionString = null;
+        try {
+            String output = getHttpsServlet("/metrics");
+            Log.info(c, "externalPrometheusMicrometer", output);
+            Assert.assertNotNull(output);
+
+            //just do simple check for jvm.uptime metric
+            boolean containsMetrics = (output.contains("jvm_uptime_seconds{mp_scope=\"base\",")) ? true : false;
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.5.0_fat/fat/src/io/openliberty/microprofile/metrics50/internal/tck/launcher/LibraryRefTest.java]: Just need `boolean containsMetrics = output.contains("jvm_uptime_seconds{mp_scope=\"base\","));`

**Review by fmhwong — DISMISSED** [review]
LGTM with some minor comments:

1. The contributors line in the copyright block is not needed anymore.
2. Some assert statements have no messages.

**Comment by Channyboy** [comment]
#build

**Review by fmhwong — DISMISSED** [review]
LGTM

**Comment by Channyboy** [comment]
#build

**Review by fmhwong — DISMISSED** [review]
LGTM

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

File: dev/io.openliberty.microprofile.metrics.internal.5.0_fat/fat/src/io/openliberty/microprofile/metrics50/internal/tck/launcher/LibraryRefTest.java [hunk 2]
```diff
@@ -72,6 +95,22 @@ public void after() throws Exception {
...
+    private static void isJava11014() throws IOException {
+        JavaInfo javaInfo = JavaInfo.forServer(server);
+
+        Log.info(c, "isJava11014",
+                 "Major.minor.micro : [" + JavaInfo.forServer(server).majorVersion()
+                                   + "."
+                                   + JavaInfo.forServer(server).minorVersion()
+                                   + "." + JavaInfo.forServer(server).microVersion()
+                                   + "]");
+        if (javaInfo.majorVersion() == 11 && javaInfo.microVersion() == 17)
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.5.0_fat/fat/src/io/openliberty/microprofile/metrics50/internal/tck/launcher/LibraryRefTest.java]: Is the microVersion of 11.0.14 jdk really `17`?

Comment (Channyboy) [reply dev/io.openliberty.microprofile.metrics.internal.5.0_fat/fat/src/io/openliberty/microprofile/metrics50/internal/tck/launcher/LibraryRefTest.java]: No, that was left over :(

**Comment by Channyboy** [comment]
#build

**Review by fmhwong — APPROVED** [review]
LGTM

**Comment by Channyboy** [comment]
Rebuilding. The 10 FAT executions completed successfully. But one of the forked builds which seem unrelated to spinning up the  (main) FAT runs didn't complete. 
#build

**Comment by Channyboy** [comment]
#libby

**Pull request merged**